src/java.base/share/classes/java/lang/invoke/InvokerBytecodeGenerator.java

```diff
@@ -265,27 +267,27 @@ private static MemberName resolveInvokerMember(Class<?> invokerClass, String nam
     /**
      * Set up class file generation.
      */
-    private ClassWriter classFilePrologue() {
+    private byte[] classFilePrologue(Consumer<? super ClassBuilder> config) {
```

Comment:
This should no longer be called a "prologue" as it doesn't only execute before the config; it should be called like a `classFileSetup`. Same for the method one.

Reply:
Renamed, thanks.